rsts/index.rst

```diff
@@ -94,6 +94,8 @@ Who's Using Flyte?
 
 At `Lyft <https://eng.lyft.com/introducing-flyte-cloud-native-machine-learning-and-data-processing-platform-fb2bb3046a59>`__, Flyte has served production model training and data processing for over four years, becoming the de-facto platform for the Pricing, Locations, ETA, Mapping teams, Growth, Autonomous and other teams.
 
+Flyte is also used in production by `Spotify <https://www.spotify.com/>`__, `Freenome <https://www.freenome.com/>`__, `USU Solutions <https://www.usu.com/>`__ and `Striveworks <https://striveworks.us/>`__.
```

Comment:
i dont know if we should add this, I think all this information should be in one place (maybe the README?) and we can just link to it.
Saying- 
Here is a list of users using Flyte in production

Reply:
Please check update below.